src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/BindNeverAttribute.cs

```diff
@@ -5,9 +5,15 @@
 
 namespace Microsoft.AspNetCore.Mvc.ModelBinding
 {
+    /// <summary>
+    /// Indicates that the property should be excluded from model binding.
+    /// </summary>
```

Comment:
Yes. However the MSDN docs just say "property" and not "property or all properties of this type" for the same attribute. Will update if you feel strongly.